src/assembly/package.xml

```diff
@@ -40,14 +40,20 @@
                 kafka connect hdfs connector. -->
                 <exclude>org.mortbay.jetty:*</exclude>
                 <exclude>com.sun.jersey:*</exclude>
-                <exclude>org.eclipse.jetty.aggregate:jetty-all</exclude>
-                <exclude>com.sun.jersey.contribs:jersey-guice</exclude>
                 <exclude>io.confluent:kafka-connect-storage-common</exclude>
                 <exclude>io.confluent:kafka-connect-storage-core</exclude>
                 <exclude>io.confluent:kafka-connect-storage-format</exclude>
                 <exclude>io.confluent:kafka-connect-storage-partitioner</exclude>
                 <exclude>io.confluent:kafka-connect-storage-wal</exclude>
             </excludes>
         </dependencySet>
+        <dependencySet>
```

Comment:
The original `dependencySet` entry excludes this shaded JAR by `<exclude>io.confluent:kafka-connect-storage-common</exclude>`, this PR adds one more `dependencySet`, that does `include` instead of `exclude`.